prov/verbs/src/fi_verbs.h

```diff
@@ -162,6 +173,16 @@ struct fi_ibv_cq {
 	enum fi_cq_wait_cond	wait_cond;
 	struct ibv_wc		wc;
 	int			signal_fd[2];
+	fi_ibv_cq_read_entry	read_entry;
+	struct util_buf_pool	*wce_pool;
+	struct slist		wcq;
+	fastlock_t		lock;
+	struct util_buf_pool	*epe_pool;
+	struct slist		ep_list;
+	fastlock_t		ep_list_lock;
+	uint64_t		ep_cnt;
+	uint64_t		send_signal_wr_id;
+	uint64_t		wr_id_mask;
```

Comment:
Gurgle - this is a lot of overhead...  Can the pools be moved to the domain or fabric to increase sharing?  Are 2 locks needed, or could a single lock be used?